x-pack/plugins/secops/public/components/timeline/body/renderers/auditd/index.tsx

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export * from './generic_details';
```

Comment:
General, some of these components are not used outside of this folder. so I am wondering why we are exporting them anyway. I am asking because when I saw that I was questioning myself if I miss understood something and I searched for all the component and realizing they were some not used outside of the folder.

Reply:
I find if someone forgets to export one and then later someone else uses the component they don't typically remember to add it and we get duplicate imports, so I ahead of time do the barrel rolls.